### PR TITLE
Tweak: User preferences: Added panel width control

### PR DESCRIPTION
**Pull request by kobizz**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/editor/components/settings/editor-preferences/manager.js [hunk 1]
```diff
@@ -52,6 +53,12 @@ export default class extends BaseManager {
 		$link.attr( 'media', 'auto' === newValue ? '(prefers-color-scheme: dark)' : '' ).appendTo( elementorCommon.elements.$body );
 	}
 
+	onPanelWidthChanged( newValue ) {
+		elementor.panel.saveSize( { width: newValue.size + newValue.unit } );
+
+		elementor.panel.setSize();
```
Comment (matipojo) [inline assets/dev/js/editor/components/settings/editor-preferences/manager.js:59]: It's already set, no?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/components/settings/editor-preferences/manager.js (hunk 1 above)
Comment (matipojo) [reply assets/dev/js/editor/components/settings/editor-preferences/manager.js:59]: @kobizz

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/components/settings/editor-preferences/manager.js (hunk 1 above)
Comment (kobizz) [reply assets/dev/js/editor/components/settings/editor-preferences/manager.js:59]: No. This method just applying the size to the panel width

Comment (matipojo) [reply assets/dev/js/editor/components/settings/editor-preferences/manager.js:59]: Can you rename it to applySize?

Comment (kobizz) [reply assets/dev/js/editor/components/settings/editor-preferences/manager.js:59]: This is an old method. We can't deal with backward compatibility at this stage

**Review by matipojo — APPROVED** [review]

**Pull request merged**